Detections/ASimAuthentication/imAuthBruteForce.yaml

```diff
@@ -37,10 +37,10 @@ entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
-        columnName: AccountCustomEntity
+        columnName: TargetUsername
   - entityType: IP
     fieldMappings:
       - identifier: Address
-        columnName: IPCustomEntity
-version: 1.1.0
-kind: Scheduled
\ No newline at end of file
+        columnName: IPAddress
```

Comment:
Should be SrcDvcIpAddr

Reply:
Column should now have the correct value.